.github/workflows/test.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
     - run: npm install
-    - run: npm start & sleep 5 && npm test
+    - run: npm test
```

Comment:
<img width="1661" alt="image" src="https://user-images.githubusercontent.com/38065883/167934018-d862d4f5-64ca-4578-a4f5-2ba6418a6fa6.png">
gaps between cards should be wider

Reply:
https://user-images.githubusercontent.com/38065883/167934093-ea991803-8a59-4c10-9a7d-af626ffcfea7.mov

strange :)

Reply:
<img width="1680" alt="image" src="https://user-images.githubusercontent.com/38065883/167934929-eb1a8ec4-5fc5-4932-a023-fb8ec9a22f10.png">
Form is too wide, take a look how much columns it takes in design

Reply:
<img width="494" alt="image" src="https://user-images.githubusercontent.com/38065883/167935239-1a459989-1fda-4a7e-9dd0-1548c7d533d4.png">